components/TextFields/src/FontThemer/MDCTextFieldFontThemer.h

```diff
@@ -0,0 +1,58 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+
+@class MDCTextField;
+
+@protocol MDCFontScheme;
+@protocol MDCTextInputController;
+
+/**
+ Used to apply a font scheme to theme to MDCTextField/MDCTextInputController.
+ */
+@interface MDCTextFieldFontThemer : NSObject
+
+/**
+ Applies a font scheme to theme to a MDCTextInputController instance.
+
+ @param fontScheme The font scheme to apply to a MDCTextInputController.
+ @param textInputController A MDCTextInputController instance to apply a font scheme.
+ */
++ (void)applyFontScheme:(nonnull id<MDCFontScheme>)fontScheme
+  toTextInputController:(nullable id<MDCTextInputController>)textInputController;
+
+/**
+ Applies a font scheme to theme to MDCTextInputController for a specific class type. Would not apply
```

Comment:
`Will not apply` would sound a little better.

Reply:
`Applies a font scheme to theme to MDCTextInputController` is an unclear phrase. That's what I meant in the comment from before.

Reply:
I changed the phrasing.